content/millennium/r4/financial/general/financial-transaction.md

```diff
@@ -44,6 +45,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `financial-transaction-method`          | [`string`]                        | Describes the method of payment for the financial transaction.
  `financial-transaction-tendered-amount` | [`Money`]                         | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
  `financial-transaction-type`            | [`CodeableConcept`]               | The classification of the transaction.
+ `apply-first-in-first-out`              | [`boolean`]                       | Whether to apply FIFO logic for recurring encounters or not.
```

Comment:
I believe these should be in alphabetical order.

Reply:
Addressed in 10f7519981f3a6a5b0220a264d546cce6fc7fed5